thijsvanloef/palworld-server-docker
# Updating configmap

## Context 

* Now the new server config environment variables are supported;
* The new game world environment variables are supported;

## Choices

* Following up the docker image updates, not the most elegant solution, but checking and formatting values will be for now only for the server settings.
* The game world settings are not validated in anyway, just injected on the container as-is;

## Test instructions

Git clone the repo;
Install Helm client and check if you have access to a Kubernetes cluster (tested on v1.29);
Create your values.override.yaml file and add the properties that are pertinent to your cluster;
Run the following command:
helm install --create-namespace --namespace palworld palworld chart/ --values values.override.yaml

## Checklist before requesting a review
